### PR TITLE
Fix offers item size

### PR DESCRIPTION
**Pull request by OrenZak**
#### Main purpose:
Handle better the image ratio
Earn images should be 420X300
Spend images should be 900X300
#### Technical description:
 - calculate height according to width on screen

### CODE REVIEW

File: sdk/src/main/java/com/kin/ecosystem/marketplace/view/EarnRecyclerAdapter.java [hunk 1]
```diff
@@ -1,10 +1,29 @@
...
+
+	private static final float IMAGE_WIDTH_XX_RES_RATIO = 0.333f;
+	private static final float IMAGE_HEIGHT_XX_RES_RATIO = 0.664f;
+
     EarnRecyclerAdapter() {
         super(R.layout.kinecosystem_earn_recycler_item);
     }
+
+	@Override
+	protected float getImageWidthToScreenRatio() {
```
Comment (YohayBar) [inline sdk/src/main/java/com/kin/ecosystem/marketplace/view/EarnRecyclerAdapter.java]: why diffrent method naming  getImageWidthToScreenRatio and getImageHeightRatio (i.e the ToScreen in the width and not in Height)

File: sdk/src/main/java/com/kin/ecosystem/marketplace/view/EarnRecyclerAdapter.java [hunk 2]
```diff
@@ -1,10 +1,29 @@
 package com.kin.ecosystem.marketplace.view;
 
+import static com.kin.ecosystem.core.util.DeviceUtils.DensityDpi.XXHDPI;
+
 import com.kin.ecosystem.R;
+import com.kin.ecosystem.core.util.DeviceUtils;
 
 class EarnRecyclerAdapter extends OfferRecyclerAdapter {
 
+    private static final float IMAGE_WIDTH_RATIO = 0.388f;
```
Comment (YohayBar) [inline sdk/src/main/java/com/kin/ecosystem/marketplace/view/EarnRecyclerAdapter.java]: IMAGE_WIDTH_RATIO  -> IMAGE_WIDTH_TO_SCREEN_RATIO
IMAGE_WIDTH_XX_RES_RATIO -> IMAGE_WIDTH_XX_RES_TO_SCREEN_RATIO

File: sdk/src/main/java/com/kin/ecosystem/marketplace/view/SpendRecyclerAdapter.java [hunk 3]
```diff
@@ -1,17 +1,29 @@
...
+import static com.kin.ecosystem.core.util.DeviceUtils.DensityDpi.XXHDPI;
+
 import com.kin.ecosystem.R;
+import com.kin.ecosystem.core.util.DeviceUtils;
 
 class SpendRecyclerAdapter extends OfferRecyclerAdapter {
 
-    private static final float IMAGE_WIDTH_RATIO = 0.8f;
+    private static final float IMAGE_WIDTH_RATIO = 0.833f;
+    private static final float IMAGE_HEIGHT_RATIO = 0.333f;
```
Comment (YohayBar) [inline sdk/src/main/java/com/kin/ecosystem/marketplace/view/SpendRecyclerAdapter.java:11]: Same width should be named with TO_SCREEN

File: core/src/main/java/com/kin/ecosystem/core/util/DeviceUtils.java [hunk 4]
```diff
@@ -43,7 +43,7 @@ private static void checkDensityDpi(DisplayMetrics displayMetrics) {
         int dpi = displayMetrics.densityDpi;
         if (dpi <= DENSITY_HIGH) {
             densityDpi = HDPI;
-        } else if (dpi >= DENSITY_XHIGH && dpi <= DENSITY_XXHIGH) {
+        } else if (dpi >= DENSITY_XHIGH && dpi < DENSITY_XXHIGH) {
```
Comment (YohayBar) [inline core/src/main/java/com/kin/ecosystem/core/util/DeviceUtils.java:46]: ">=" should be ">" as "<=" is used above (doesn't really matter :) )

**Review by YohayBar — APPROVED** [review]
see my very very minor comments otherwise 👍

File: core/src/main/java/com/kin/ecosystem/core/util/DeviceUtils.java (hunk 4 above)
Comment (OrenZak) [reply core/src/main/java/com/kin/ecosystem/core/util/DeviceUtils.java:46]: it's not

**Pull request merged**